larray/core/axis.py

```diff
@@ -1708,6 +1708,15 @@ def index(self, axis):
 
                 # 3) otherwise look for any axis with the same name
                 name = axis.name
+                # try:
+                #     # 3) otherwise look for any axis with the same name
+                #     return self.names.index(axis.name)
+                # except ValueError:
+                #     # 4) try any compatible axis
+                #     for i, item in enumerate(self._list):
+                #         if item.iscompatible(axis):
+                #             return i
+                # raise ValueError("%r is not in collection" % axis)
```

Comment:
remove code in comment ?

Reply:
Yes. This was an experiment I forgot to remove. I think we will need something along those lines later, but this needs more thoughts.